server/routes/referrals/complexityLevelPresenter.test.ts

```diff
@@ -61,15 +39,15 @@ describe('ComplexityLevelPresenter', () => {
     describe('when the referral already has a selected complexity level', () => {
       it('sets checked to true for the referral’s selected complexity level', () => {
         draftReferral.complexityLevelId = '110f2405-d944-4c15-836c-0c6684e2aa78'
```

Comment:
To decrease dependence on the exact values in the factory:

```suggestion
        draftReferral.complexityLevelId = serviceCategory.complexityLevels[1].id
```